base/applications/taskmgr/graphctl.c

```diff
@@ -342,6 +352,8 @@ void GraphCtrl_InvalidateCtrl(TGraphCtrl* this, BOOL bResize)
             DeleteObject(this->m_bitmapPlot);
             this->m_bitmapPlot = CreateCompatibleBitmap(dc, this->m_nClientWidth, this->m_nClientHeight);
             SelectObject(this->m_dcPlot, this->m_bitmapPlot);
+            printf("try redraw\n");
```

Comment:
Is it worth having this debug print message?

Reply:
OMG, I forgot deleting it. It was printf debugging. :sweat:

Reply:
Deleted.